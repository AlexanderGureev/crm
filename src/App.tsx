import React from 'react';
import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
import { Home } from './screens/Home';
import { Dashboard } from './screens/Dashboard';
import { PrivateRoute } from './components/PrivateRoute';

const App: React.FC = () => {
  return (
    <Router>
      <Switch>
        <Route exact path="/" component={Home} />
        <PrivateRoute path="/dashboard" component={Dashboard} />
      </Switch>
    </Router>
  );
};

export default App;
