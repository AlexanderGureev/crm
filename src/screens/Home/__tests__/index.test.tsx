import { shallow } from 'enzyme';
import React from 'react';
import App from '../../../App';

describe('App', () => {
  it('app renders correctly', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
