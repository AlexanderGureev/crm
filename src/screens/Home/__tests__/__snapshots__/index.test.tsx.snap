// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App app renders correctly 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <PrivateRoute
      component={[Function]}
      path="/dashboard"
    />
  </Switch>
</BrowserRouter>
`;
